src/core/qgsunittypes.cpp

```diff
@@ -1256,6 +1256,8 @@ double QgsUnitTypes::fromUnitToUnitFactor( QgsUnitTypes::AngleUnit fromUnit, Qgs
           return 3600;
         case AngleTurn:
           return 1.0 / 360.0;
+        case AngleMil:
```

Comment:
These ones will need to be added to existing angle unit tests

Reply:
Done, some expected failures in their current form

Reply:
Done, any more tests to do?